modules/pensions/lib/pdf_fill/forms/va21p527ez.rb

```diff
@@ -13,6 +13,10 @@ class Va21p527ez < FormBase
       include FormHelper::PhoneNumberFormatting
       include ActiveSupport::NumberHelper
 
+      FORM_ID = '21P-527EZ'
+
+      TEMPLATE = 'modules/pensions/lib/pdf_fill/forms/pdfs/21P-527EZ.pdf'
```

Comment:
is it possible to get the 'path' of the module programmatically, so it could be something like 
`MODULE_PATH + /lib/pdf_fill/...`?